### ISSUE TITLE
More prominent install action on item pages

### ISSUE THREAD

**Issue by nedrichards** [flathub/frontend#179]
![Screenshot from 2022-01-31 20-53-10](https://user-images.githubusercontent.com/302677/151871407-8fb95b5c-2538-4523-8765-5c46b41d0457.png)

Our primary goal is to convert web browsers into performing a positive action on an item page (eg. install, donate, contribution). We have a *lot* of space in the header with quite small action areas. Even for long app titles like "PulseAudio Volume Control" there's plenty of space.

We should move the donate and install buttons to side by side rather than on top of each other and double the size.

**Comment by razzeee** [comment]
This will break the mobile view. Which could be worked around. Still, I'm not sure that will look nice at all. Especially with gigantic buttons, alignment might also be a problem. We need these to balance out the app icon on the left. (which already might be a problem with pages without donate buttons)

**Comment by nedrichards** [comment]
I'll have a closer look at the mobile specific views. In some very rough testing it 'seemed like it would be fine on my device', which is obviously famous last words. Primary usage for installs is desktop of course, by a long, long way. Fine to hold until some more detailed design occurs.

**Comment by razzeee** [comment]
If you say fine to hold, we should probably use github labels to track something like this. Otherwise I won't be able to keep track.

I think I tested with the lowest available resolution in firefox responsive design mode which is 320x480, but yeah, really depends on what we want and how relevant linux mobile devices will be in the next years.

**Comment by nedrichards** [comment]
Agreed on the labels. I'm actually thinking about using the milestones as well, that's going to be next 'weekend work'.
